src/__tests__/index.test.ts

```diff
@@ -961,7 +961,7 @@ const vectors: Array<TestVector> = [
     name: 'IOTA',
     coinType: 4218,
     passingVectors: [
-      { text: 'Qc6iYCZWn4BauKXGYirRG8pMtgdHMk2dzn', hex: '3aa9f8f3b055324f6b2d6bcac328ec2d7e3cd22d8b' },
+      { text: 'iota1qpw6k49dedaxrt854rau02talgfshgt0jlm5w8x9nk5ts6f5x5m759nh2ml', hex: '005dab54adcb7a61acf4a8fbc7a97dfa130ba16f97f7471cc59da8b869343537ea' },
```

Comment:
Why is there `00` before the actual address?
It should be `5dab54adcb7a61acf4a8fbc7a97dfa130ba16f97f7471cc59da8b869343537ea` alone I think

Reply:
when I tested the encoder/decoder it output with the '00..' and resolved to the correct 'iota..' address

Reply:
@admljhnsn the nodes will return an error with this added
https://chrysalis-nodes.iota.org/api/v1/addresses/ed25519/005dab54adcb7a61acf4a8fbc7a97dfa130ba16f97f7471cc59da8b869343537ea
`{"error":{"code":"400","message":"invalid parameter, error: invalid address length: 005dab54adcb7a61acf4a8fbc7a97dfa130ba16f97f7471cc59da8b869343537ea: code=400, message=invalid parameter"}}`
https://chrysalis-nodes.iota.org/api/v1/addresses/ed25519/5dab54adcb7a61acf4a8fbc7a97dfa130ba16f97f7471cc59da8b869343537ea
`{"data":{"addressType":0,"address":"5dab54adcb7a61acf4a8fbc7a97dfa130ba16f97f7471cc59da8b869343537ea","balance":0,"dustAllowed":false,"ledgerIndex":1075944}}`

Reply:
@Thoralf-M dang, i barely figured out how to get this implemented, i don't know how to fix that here 😢. i guess it has something to do with how the bech32Chain() function works. i'm a dev noob so i don't know what to do

Reply:
The hex(33 bytes) from bech32Chain() is correct, the Chrysalis address consists of version and address parts, it has a leading version byte for ED25519(0x00) and WOTS(0x01). `addresses/ed25519` only accept ed25519 address without the version field. 
see [RFC: Bech32 Address Format](https://github.com/Wollac/protocol-rfcs/blob/bech32-address-format/text/0020-bech32-address-format/0020-bech32-address-format.md)